Modelica/Resources/Reference/Modelica/Clocked/Examples/Elementary/RealSignals/HoldWithDAeffects2/comparisonSignals.txt

```diff
@@ -1,4 +1,4 @@
 time
-sample.y
-hold.y
-shiftSample.y
+sample1.y
+hold1.y
+shiftSample1.y
```

Comment:
```suggestion
// shiftSample1.y
```

According to https://github.com/modelica/ModelicaStandardLibrary/issues/3516#issuecomment-607709487 shiftSample1.y must not be used as comparison signal.